subprojects/junit5/src/main/java/org/mockito/junit5/WithMockito.java

```diff
@@ -0,0 +1,28 @@
+package org.mockito.junit5;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.MockitoRule;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+
+/**
+ * Place this annotation on your root test class in order to initialize fields annotated with {@link Mock @Mock},
+ * {@link Spy @Spy} or {@link Captor @Captor}. To specify the stubbing strictness you can declare the
+ * annotation {@link Strictness @Stricness}
+ * <p>
+ * This is the Junit5 equivalent to {@link MockitoRule} and {@link MockitoJUnitRunner} for JUnit4.
+ */
+@Retention(RUNTIME)
+@Target({ANNOTATION_TYPE, TYPE})
+@ExtendWith(MockitoExtension.class)
+public @interface WithMockito {}
```

Comment:
We have to bike-shed on this name. `WithMockito` is vague, but don't have a concrete proposal for rename yet